sdks/python/build.gradle

```diff
@@ -456,10 +453,39 @@ project.task('crossLanguagePythonJava') {
   dependsOn 'setupVirtualenv'
   dependsOn ':beam-sdks-java-container:docker'
   dependsOn ':beam-runners-core-construction-java:buildTestExpansionServiceJar'
+
   doLast {
+    def testServiceExpansionJar = project(":beam-runners-core-construction-java:").buildTestExpansionServiceJar.archivePath
+    exec {
+      executable 'sh'
+      args '-c', ". ${project.ext.envdir}/bin/activate && pip install -e .[test] && python -m apache_beam.transforms.external_test --expansion_service_jar=${testServiceExpansionJar}"
+    }
+  }
+}
+
+project.task('crossLanguagePortableWordCount') {
+  dependsOn 'setupVirtualenv'
+  dependsOn ':beam-runners-flink_2.11-job-server-container:docker'
+  dependsOn ':beam-sdks-python-container:docker'
+  dependsOn ':beam-sdks-java-container:docker'
+  dependsOn ':beam-runners-core-construction-java:buildTestExpansionServiceJar'
+
+  doLast {
+    def testServiceExpansionJar = project(":beam-runners-core-construction-java:").buildTestExpansionServiceJar.archivePath
```

Comment:
Note that the ExpansionService has moved from`core-construction-java` to `:java-fn-execution`

Reply:
Thanks for your comment. IMHO, `core-construction-java` is the better place for `ExpansionService` since `ExpansionService` mainly works during the pipeline construction phase. And, because `java-fn-execution` depends on `core-construction-java`, we can still bundle `ExpansionService` with `JobServer` without moving it. When we put `ExpansionService` in `java-fn-execution`, it will block us from creating a class in `core-construction-java` which uses `ExpansionService` class (e.g., external transform for Java SDK and its test).

Reply:
I agree, I will move it back. See my comment in #7947.